index.html

```diff
@@ -2,24 +2,13 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <link rel="stylesheet.css"/>
-        <title>gatehopper</title>
+        <link rel="stylesheet" type="text/css" href="stylesheet.css"/>
+        <title>Gatehopper</title>
```

Comment:
There seems to be a lot of noise in this PR - not sure why these changes are showing up in this diff, as they're already on `main`. Is this something you would be able to investigate and remedy? That would make it easier to review, so we're not hunting through the diff to see what changes are actually being introduced.

Reply:
It might be because I started it before the refactor and now we would be merging it in after the refactor. I might merge again once more to clean it up, especially now that I am using meld rather than a text editor to resolve conflicts.